client/src/components/TrailShowPage.js

```diff
@@ -1,12 +1,18 @@
 import React, { useState, useEffect } from "react"
+import ReviewForm from "./ReviewForm"
+import ErrorList from "./ErrorList"
+import translateServerErrors from "../services/translateServerErrors"
 
 const TrailShowPage = (props) => {
   const [trail, setTrail] = useState({
     reviews: [],
   })
+  const [newReview, setNewReview] = useState({})
```

Comment:
Don't need this additional piece of state, this implementation will cause extra re-renders of this component

Reply:
done deleted.